### ISSUE TITLE
Training on custom dataset

### ISSUE THREAD

**Issue by nviolante25** [AustinXY/GIRAFFEHD#6]
Hello,
Thanks for the repo!

I'm trying to train GIRAFFE HD on a version of CompCar with the removed background at 256x256 resolution. Based on what I read in [this other issue](https://github.com/AustinXY/GIRAFFEHD/issues/2#issuecomment-1295256945), I trained the model for 50k iterations on 8 GPUs with batch size 32, which should be equivalent in terms of total images. However, my generated images are not looking good. These are some examples of my last checkpoint

![049500_0](https://github.com/AustinXY/GIRAFFEHD/assets/41066109/afc0bce4-f3ab-457a-856e-4270bf6b0370)

Do you think this is a problem with the number of iterations and I should continue training for a longer time or is something else in the training config as well?

Thanks in advance!

**Comment by AustinXY** [comment]
Hello,

Sorry for the late reply. Based on your description and the images you've provided, it seems that the results are indeed not as good as expected.

One possible problem could stem from the way you're removing the background from your dataset. If you're utilizing a binary mask for this task, it could be inadvertently providing the discriminator with a clear signal. This happens because the generator would be struggling to mimic the sharp edges created by the mask, thereby tipping the balance of the training process in favor of the discriminator.

You can confirm if this is indeed the case by monitoring the training loss. If the discriminator's loss is significantly low, it likely indicates that this is the root of the problem.

**Comment by nviolante25** [comment]
Hello,
Thank you for your response! I'll check that and maybe try to train substituting the background generator with a constant black background.
